### PR TITLE
work complete

### PR DESCRIPTION
**Pull request by AvigailGellis**
Nice scenario! concise and clear. Here is the completed work, let me know if there are any corrections.

### CODE REVIEW

File: MedicalSuppliesDB/4 reports.sql [hunk 1]
```diff
@@ -0,0 +1,28 @@
+--How many times each item was borrowed, ordered by most often to least often.
+select TotalBorrowed = count(*), g.Item
+from GemachCustomer g 
+group by g.Item
+order by TotalBorrowed desc
+--A list of all items sorted by the condition they're currently in.
+--AS This wasn't so clear, is this what you wanted?
```
Comment (ChanaCohen3896) [inline MedicalSuppliesDB/4 reports.sql:7]: That's fine.

File: MedicalSuppliesDB/4 reports.sql [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+--For items that haven't been returned yet, how long has it been since they were borrowed?
+select g.Item, DaysSinceBorrowed = datediff(day, g.DateBorrowed, getdate())
+from GemachCustomer g 
+where g.DateReturned is null
+--I want to know how many people gave donations and how many declined.
+select NoDonation = count(*)
+from GemachCustomer g 
+where g.Donation = 0 and g.DateReturned is not null
+
+select NoDonation = count(*)
```
Comment (ChanaCohen3896) [inline MedicalSuppliesDB/4 reports.sql]: The results of both select statement are called NoDonation, the second one should be called DonationMade or something like that.

**Review by ChanaCohen3896 — CHANGES_REQUESTED** [review]
It looks really good! I found only one small mistake.

**Review by ChanaCohen3896 — APPROVED** [review]

File: MedicalSuppliesDB/4 reports.sql (hunk 2 above)
Comment (AvigailGellis) [reply MedicalSuppliesDB/4 reports.sql]: fixed

**Pull request closed without merging**